pnpm-lock.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
     version: 1.0.0
   '@warp-ds/css':
     specifier: ^1.0.0
-    version: 1.0.0
+    version: link:../css-1
```

Comment:
oopsy 😁

Reply:
Ah, the css PR needs to be out first! Will have a look there.

Reply:
Classic 🙄 Fixed